index.html

```diff
@@ -19,28 +19,167 @@ <h4>Yunus Ali</h4>
+                            <span class="year"> &nbsp;&nbsp; &#8226; &nbsp;&nbsp; Back End Dev &nbsp;&nbsp; &#8226; &nbsp;&nbsp;2015</span>
+                          </article>
+                        <p class="description">Hi there! My name is Anna and I am a book lover, traveler and professional blogger. Follow me to stay connected!</p>
+                        <ul class="stacks">
+                            <li class="language">html</li>
+                            <li class="language">css</li>
+                            <li class="language">javaScript</li>
+                          </ul>
+                        <button class='btn' type="button">see projects</button>
+                      </div>
+
+
+                      <div class="container">
+                        <img src="assets/Portfolio3.png" alt="">
+                        <h1 class="Projeect-Title">Tonic</h1>
+                        <article class="experience">
+                            <span class="title">CANOPY</span>
+                            <span class="year"> &nbsp;&nbsp; &#8226; &nbsp;&nbsp; Back End Dev &nbsp;&nbsp; &#8226; &nbsp;&nbsp;2015</span>
+                          </article>
+                        <p class="description">Hi there! My name is Anna and I am a book lover, traveler and professional blogger. Follow me to stay connected!</p>
+                        <ul class="stacks">
+                            <li class="language">html</li>
+                            <li class="language">css</li>
+                            <li class="language">javaScript</li>
+                          </ul>
+                        <button class='btn' type="button">see projects</button>
+                      </div>
+
+
+                      <div class="container">
+                        <img src="assets/Portfolio4.png" alt="">
+                        <h1 class="Projeect-Title">Multi-Post
+                          Stories</h1>
+                        <article class="experience">
+                            <span class="title">CANOPY</span>
+                            <span class="year"> &nbsp;&nbsp; &#8226; &nbsp;&nbsp; Back End Dev &nbsp;&nbsp; &#8226; &nbsp;&nbsp;2015</span>
+                          </article>
+                        <p class="description">Hi there! My name is Anna and I am a book lover, traveler and professional blogger. Follow me to stay connected!</p>
+                        <ul class="stacks">
+                            <li class="language">html</li>
+                            <li class="language">css</li>
+                            <li class="language">javaScript</li>
+                          </ul>
+                        <br>
+                        <button class='btn' type="button">See projects</button>
+                      </div>
+
+            </section>
+            <section class="aboutme">
+
+                <article class="aboutme-article">
+                    <h1>About Myself</h1>
```

Comment:
- [ optional] you can break the `about myself` using `<be>` so it looks like the figma version

Reply:
About <br>
Myself

like the figma design... its optional though